launcher/verifier/rest/rest.go

```diff
@@ -157,6 +157,14 @@ func convertRequestToREST(request verifier.VerifyAttestationRequest) *v1alpha1.V
 	}
 }
 
+func handleError(err error, message string) error {
+	const invalidArgument = "invalid argument"
+	if strings.Contains(err.Error(), invalidArgument) {
```

Comment:
Can we instead parse the error as a *googleapi.Error and use the appropriate status code? For example, see the ProjectsLocationsGetCall.Do documentation for our v1alpha1 client https://github.com/josephlr/google-api-go-client/blob/main/confidentialcomputing/v1alpha1/confidentialcomputing-gen.go#L589-L595

and an example in our v1 client: https://github.com/googleapis/google-cloud-go/blob/main/confidentialcomputing/apiv1/confidential_computing_client.go#L613

https://github.com/googleapis/googleapis/blob/master/google/rpc/code.proto